drivers/Makefile.dep

```diff
@@ -199,3 +199,9 @@ ifneq (,$(filter xbee,$(USEMODULE)))
   USEMODULE += xtimer
   USEMODULE += netif
 endif
+
+ifneq (,$(filter uart_half_duplex,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_gpio
```

Comment:
indeed, there are inconsistencies about indentation. Maybe that should deserve a dedicated PR (make indentation consistent in the whole Makefile.dep)